docs/src/docs/asciidoc/user/usermanaged.adoc

```diff
@@ -25,36 +25,52 @@ Of course, if you find yourself requiring plenty of services, maybe the cache ma
 
 === API extensions
 
-While a `UserManagedCache` extends `Cache`, it offers additional methods:
+If you use a `UserManagedCache`, you need to configure all required services by hand.
+The `UserManagedCache` class extends the `Cache` class by offering additional methods:
+
+* init() - initializes the cache
```

Comment:
The methods listed here should be changed to code font by surrounding them with backticks.